src/read.rs

```diff
@@ -1138,11 +1138,11 @@ where
 {
     match tri!(read.peek()) {
         Some(b) => Ok(b),
-        None => error(read, ErrorCode::EofWhileParsingString),
+        None => error_no_replace(read, ErrorCode::EofWhileParsingString),
     }
 }
 
-fn error<'de, R, T>(read: &R, reason: ErrorCode) -> Result<T>
+fn error_no_replace<'de, R, T>(read: &R, reason: ErrorCode) -> Result<T>
 where
```

Comment:
At some point in the future, a stupid person, most probably me, will have to be resolving a merge conflict and will wonder "why isn't there an error function in serde_json_lenient"? I'll forget all this discussion ever happened.

To make life easy for stupid me, please would you re-add a dummy `error` function but make it clear it's Very Bad? Either a runtime panic, or better still, use [these tricks to get a static compile-time panic if it's ever actually called](https://github.com/google/autocxx/blob/main/src/reference_wrapper.rs#L198). Then Stupid Person will be able to follow the breadcrumbs to your nice code comment which says to use `error_no_replace` instead.

Reply:
It's probably easier to just change its signature :)